### PR TITLE
refactor: remove unused host elements from components.

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** N/A

## Summary

refactor: remove unused host elements from components.

Uses <Fragment> when necessary. Also runs prettier.

### CODE REVIEW

**Review by eriklharper — APPROVED** [review]

File: src/components/calcite-combobox/calcite-combobox.tsx [hunk 1]
```diff
@@ -885,7 +885,7 @@ export class CalciteCombobox {
     const single = this.selectionMode === "single";
     const labelId = `${guid}-label`;
     return (
-      <Host>
+      <Fragment>
```
Comment (jcfranco) [inline src/components/calcite-combobox/calcite-combobox.tsx:888]: Should we keep `Host` when there's more than one child? Using `Fragment` doesn't reduce nesting and I'm not sure what we gain here.

Comment (driskull) [reply src/components/calcite-combobox/calcite-combobox.tsx:888]: Yeah, I guess we just protect from modifying the host.

Comment (jcfranco) [reply src/components/calcite-combobox/calcite-combobox.tsx:888]: Cool. We've got the new ESLint rules for that too, so I think the combo will help keep the host clean.

**Review by jcfranco — APPROVED** [review]
👍🏼

**Pull request merged**